src/constants.js

```diff
@@ -28,4 +28,5 @@ export const errorTypes = {
   "missing-key": "Key is missing",
   "invalid-header": "Header is not valid",
   "invalid-json": "Json format is not valid",
+  "dupliated-key": "Key/Keys are not unique"
```

Comment:
Typo '`dupliated-key`'

```suggestion
  "duplicate-key": "Key/Keys are not unique"
```

Reference: https://english.stackexchange.com/questions/67463/duplicate-data-or-duplicated-data

Reply:
This error has been fixed